train.py

```diff
@@ -189,13 +194,14 @@ def dev_step(x1_batch, x2_batch, y_batch):
                              siameseModel.input_y: y_batch,
                              siameseModel.dropout_keep_prob: FLAGS.dropout_keep_prob,
             }
-        step, loss, dist, summary = sess.run([global_step, siameseModel.loss, siameseModel.distance, summaries_merged],  feed_dict)
+        step, loss, dist, summary, out1, out2 = sess.run([global_step, siameseModel.loss, siameseModel.distance, summaries_merged,,siameseModel.out1,siameseModel.out2],  feed_dict)
         time_str = datetime.datetime.now().isoformat()
         d=compute_distance(dist, FLAGS.loss)
         correct = np.sum(y_batch==d)
+	#save out1,ou2, also reshape them to get 2 x [seq length,lstm output of one sequence] arrangement	
         #print("DEV {}: step {}, loss {:g}, acc {:g}".format(time_str, step, loss, correct))
         #print(y_batch, dist, d)
-        return summary, correct, loss
+        return summary, correct, loss, out1, out2
```

Comment:
or better will be to just save these "out1, out2" to some file (of course ordering will be an issue). Later load them and run tsne.